### DISCUSSION TITLE
Yeni gorev

### DISCUSSION THREAD

**Discussion by Ecenurcelik** [IBAX-io/go-ibax#1711] [Bug Submission]
Yeni görev açıklanmış ama sistemsel olarak bir arızanız mevcut . Retweet görevi yapılıyor ama bir sorun oluştu uyarısı alınıyor . insanlar mağdur oluyor lütfen yardımcı olun .

**Comment by murselbogac0** [comment]
Sistemsel bir sorun gün içinde hallolur

**Comment by murselbogac0** [comment]
Düzelir tekrar

**Comment by Zeynel26477575** [comment]
heralde 1-2 saat sonra tekrar denemek lazım.

**Reply by alisabanoglu** [reply]
bir saat denedim yine olmuyor. ne cep telefonundan ne de bilgisayardan devamlı aynı hatayı alıyorum...
